src/main/java/cryptator/specs/ICryptaNode.java

```diff
@@ -24,42 +24,44 @@ public interface ICryptaNode {
 	 * @return the operator
 	 */
 	CryptaOperator getOperator();
-	
+
 	/**
 	 * Gets the word associated to the node.
 	 *
 	 * @return the word
 	 */
 	char[] getWord();
-	
+
 	/**
 	 * Gets the left child if any.
 	 *
 	 * @return the left child
 	 */
 	ICryptaNode getLeftChild();
 	//FIXME Use java.util.Optional ? 
-	
+
 	/**
 	 * Gets the right child if any.
 	 *
 	 * @return the right child
 	 */
 	ICryptaNode getRightChild();
 	//FIXME Use java.util.Optional ? 
-	
+
 	/**
 	 * Checks if the node is a leaf of the tree.
 	 *
 	 * @return true, if it is leaf
 	 */
 	boolean isLeaf();
-	
+
 	/**
 	 * Checks if the node is an internal node of the tree.
 	 *
 	 * @return true, if it is internal node
 	 */
 	boolean isInternalNode();
-	
+
+	boolean isConstant();
```

Comment:
I believe that it is the best alternative. 
You can read this article that discusses a related problem:
https://dzone.com/articles/using-instanceof-and-alternatives-within-java